models/public/mosaic/Mosaic.md

```diff
@@ -0,0 +1,102 @@
+# mosaic
+
+## Use Case and High-Level Description
+
+The `mosaic` model is one of the style transfer models designed to mix the content of an image with the style of another image. The model uses the method described in [Perceptual Losses for Real-Time Style Transfer and Super-Resolution](https://arxiv.org/abs/1603.08155) along with [Instance Normalization](https://arxiv.org/pdf/1607.08022.pdf).
+
+
+## Example
+
+
+## Specification
+
+| Metric            | Value            |
+|-------------------|------------------|
+| Type              | Style Transfer   |
+| GFLOPs            | -                |
+| MParams           | -                |
+| Source framework  | ONNX\*           |
```

Comment:
The original framework seems to be PyTorch.

Reply:
Changed to PyTorch

Reply:
Please capitalize it correctly.